easybuild/easyconfigs/j/Java/Java-11.0.8-aarch64.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '11.0.8'
+versionsuffix = '-aarch64'
+
+homepage = 'http://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-%(version)s%2B10/']
+sources = ['OpenJDK11U-jdk_%(arch)s_linux_%(version)s_10.tar.gz']
+# source_urls = ['https://download.java.net/java/GA/jdk%(version_major)s/9/GPL/']
```

Comment:
Please remove the commented line and add a checksum

Also: Is the `%(arch)s` template really useful given this is already specific to aarch64? And isn't `arch=AArch64`? The filename needs that lowercased. Do we have `arch_lower`?

Reply:
Thanks! Fixes incoming.